src/napari_live_recording/common/__init__.py

```diff
@@ -4,6 +4,54 @@
 from functools import total_ordering
 import pymmcore_plus as mmc
 import os
+from qtpy.QtCore import QSettings, Qt
+import functools, pims
+
+settings = QSettings("IPHT", "Napari-Live-Recording")
+filtersDict = {}
+
+
+class Settings:
```

Comment:
Could you change the name of this class to FilterSettings or something more fitting? Maybe `FiltersConfiguration`

Reply:
Yes, I could do that. I thought we could use this class also to save other things, such as values of comboboxes etc., to make the use of the plugin for the user more convient on restart. That's why I kept the name general....